packages/ui/src/domainModules/collectionMammals/components/RecordNavigationBar/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, Icon, Grid } from 'semantic-ui-react'
+
+export const RecordNavigationBar = ({
+  recordNumber,
+  totalRecords,
+  onPreviousRecord,
+  onNextRecord,
+  onShowAllRecords,
+  onNewRecord,
+}) => {
+  return (
+    <Grid textAlign="center">
+      <Grid.Column width={2}>
+        <Button
+          icon
+          onClick={() => onPreviousRecord({ recordNumber })}
+          disabled={recordNumber === 1}
```

Comment:
there are other cases when it should also be disabled, e.g. if adding a new record. so make it 
`disabled={!onPreviousRecord}`
and then the parent component will control when that function is passed. and do the same for all things that should be possible to disable